bookdown/03-train-predict.Rmd

```diff
@@ -18,11 +18,11 @@ Additionally, this chapter includes the following use-cases
 The first step is to generate the following `mlr3` objects from the [task dictionary](#tasks) and the [learner dictionary](#learners), respectively:
 
 1. The classification task
-    ```{r}
+    ```{r train-predict-task}
```

Comment:
These changes were already merged in your prior PR right?

Reply:
Yes exactly I merely merged the branch into this one before the PR was merged